src/http/server.clj

```diff
@@ -256,6 +264,9 @@
             (not (.isFile file))
             forbidden
 
+            (some (partial matches-glob? file) globs)
```

Comment:
The listing should follow the same rule.

Reply:
Not sure what this means

Reply:
@Solaxun `reply-with-listing` should build list of files following the same "exclude" rules, so we do not show listing for files that we want to exclude.